### PR TITLE
Update Chinese translation of "about" and "getting_started"

### PR DESCRIPTION
**Pull request by alaahong**
- website_and_docs/content/documentation/about
- website_and_docs/content/documentation/getting_started

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [X] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/getting_started/installing_browser_drivers.zh-cn.md [hunk 1]
```diff
@@ -1,24 +1,16 @@
 ---
-title: "Installing browser drivers"
-linkTitle: "Installing browser drivers"
+title: "安装浏览器驱动"
+linkTitle: "安装浏览器驱动"
 weight: 2
 description: >
-  Setting up your browser ready to be automated.
+设置您的浏览器用于自动化.
```
Comment (harsha509) [inline website_and_docs/content/documentation/getting_started/installing_browser_drivers.zh-cn.md]: You need to provide one tab space in here. This is the reason build got failed AFAIK!

File: website_and_docs/content/documentation/getting_started/installing_selenium_libraries.zh-cn.md [hunk 2]
```diff
@@ -3,19 +3,10 @@ title: "安装 Selenium 库"
 linkTitle: "安装 Selenium 库"
 weight: 1
 description: >
-  Setting up the Selenium library for your favourite programming language.
+为您喜爱的编程语言设置Selenium库.
```
Comment (harsha509) [inline website_and_docs/content/documentation/getting_started/installing_selenium_libraries.zh-cn.md]: Same as above!

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @alaahong !

Thanks for the PR!

There is a build failure due to missing tab spaces in a couple of places. Could you please look into it

Thanks,
Harsha

File: website_and_docs/content/documentation/getting_started/installing_selenium_libraries.zh-cn.md (hunk 2 above)
Comment (alaahong) [reply website_and_docs/content/documentation/getting_started/installing_selenium_libraries.zh-cn.md]: Sorry for the mistake. let me configure latest hugo on my local to validate that

**Review by harsha509 — APPROVED** [review]
Thank you @alaahong!

**Pull request merged**